node/src/blockchain/blockchain_interface.rs

```diff
@@ -72,14 +73,17 @@ impl Display for BlockchainError {
             InvalidAddress => Left("Invalid address"),
             InvalidResponse => Left("Invalid response"),
             QueryFailed(msg) => Right(format!("Query failed: {}", msg)),
-            UninitializedBlockchainInterface => Left("Uninitialized blockchain interface. The parameter blockchain-service-url was not set")
+            UninitializedBlockchainInterface => Left(
+                "The parameter blockchain-service-url must be \
+            set for scanners to function properly",
```

Comment:
"We can't contact the blockchain to make or receive payments because the blockchain-service-url was not specified. As a result, this Node may be delinquency-banned."

"The parameter blockchain-service-url was not set, but must be set in order to make or receive payments."

"The value for blockchain-service-url is missing, so we can't make or receive payments. You may be delinquency-banned soon."

"To avoid being delinquency-banned, you should restart the Node with a value for blockchain-service-url."